src/Material/Button.elm

```diff
@@ -71,13 +79,89 @@ import Parts exposing (Indexed, Index)
 import Material.Helpers as Helpers
 import Material.Options as Options exposing (cs, when)
 import Material.Ripple as Ripple
+import Material.Options.Internal as Internal
 
 
 
--- MODEL
+{-| Opaque link property type
+-}
+type LinkProp = LinkProp
+
+
+{-| Link properties
+-}
+type alias LinkProperty m =
+  Options.Property LinkProp m
+
+
+{-| Specifies the URL of the page the link goes to
+-}
+href : String -> LinkProperty m
+href =
+  Html.Attributes.href >> Internal.attribute
+
+
+{-| Specifies where to open the linked document.
```

Comment:
Maybe add a link to 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

for reader's unfamiliar with the target attribute